docs/reference/action_list.rst

```diff
@@ -214,11 +214,11 @@ to more field types, see `SonataDoctrineORMAdminBundle Documentation`_.
 Customizing the query used to generate the list
 -----------------------------------------------
 
```

Comment:
we should add:

```rst
.. versionadded:: 3.??

    The ``configureQuery`` method was introduced in 3.??.
```

Reply:
@OskarStark I don't see other `versionadded` in the doc.

As a developer I expect the doc talking about the last version of sonata. And if I want the doc about another version, I chose the tag on github to look at the README.

I feel like starting to add `versionadded` everywhere in the doc for each new feature/release, will create a mess.

Reply:
You are correct, but we only have 3.x so using 3.10 for example will not include the new method, thats why we should add this.

I know that others are missing, but we should do this more to me more clear for our users. Also Symfony does this anytime the deprecate or add something new

Reply:
@OskarStark Done